core/module_specifier.rs

```diff
@@ -80,3 +109,169 @@ impl PartialEq<String> for ModuleSpecifier {
     &self.to_string() == other
   }
 }
+
+#[cfg(test)]
+mod tests {
+  use super::*;
+
+  #[test]
+  fn test_resolve_from_cwd() {
+    if cfg!(target_os = "windows") {
+      let expected_url = "file:///C:/deno/tests/006_url_imports.ts";
+
+      assert_eq!(
+        ModuleSpecifier::resolve_from_cwd(r"C:/deno/tests/006_url_imports.ts")
+          .unwrap()
+          .to_string(),
+        expected_url,
+      );
+      assert_eq!(
+        ModuleSpecifier::resolve_from_cwd(r"C:\deno\tests\006_url_imports.ts")
+          .unwrap()
+          .to_string(),
+        expected_url,
+      );
+      assert_eq!(
+        ModuleSpecifier::resolve_from_cwd(r"/deno/tests/006_url_imports.ts")
+          .unwrap()
+          .to_string(),
+        expected_url
+      );
+      assert_eq!(
+        ModuleSpecifier::resolve_from_cwd(r"\tests\006_url_imports.ts")
```

Comment:
This of course only works if the cwd is somewhere on the C: drive when the tests are run.
For me this isn't the case, my deno working directory is on D:.
So maybe chdir to c: before running this test?

Reply:
You're right, probably using this bit should do the work
```rust
      let expected_url = format!(
        "file:///{}{}",
        cwd_string.as_str(),
        "tests/006_url_imports.ts"
      ).replace("\\", "/");
```